skimage/morphology/extrema.py

```diff
@@ -178,10 +181,13 @@ def h_minima(image, h, selem=None):
     neighborhood of the set.
 
     A local minimum M of depth h is a local minimum for which
-    there is at least one path joining M with a deeper minimum on which the
-    maximal value is f(M) + h (i.e. the values along the path are not
-    increasing by more than h with respect to the minimum's value) and no
-    path for which the maximal value is smaller.
+    there is at least one path joining M with an equal or lower local minimum
+    on which the maximal value is f(M) + h (i.e. the values along the path
+    are not increasing by more than h with respect to the minimum's value)
+    and no path to an equal or lower local minimum for which the maximal
+    value is greater.
```

Comment:
should this still be "smaller" rather than "greater" at the end of the paragraph?